src/main/java/com/sinthoras/visualprospecting/hooks/HooksClient.java

```diff
@@ -39,6 +39,7 @@ public void fmlLifeCycleEvent(FMLPreInitializationEvent event) {
     // load "Do your mod setup. Build whatever data structures you care about. Register recipes."
     public void fmlLifeCycleEvent(FMLInitializationEvent event) {
         super.fmlLifeCycleEvent(event);
+        MinecraftForge.EVENT_BUS.register(new HooksKey());
```

Comment:
Since `KeyInputEvent` is posted to `FMLCommonHandler#eventBus`, no need to register to `MinecraftForge.EVENT_BUS`.
```suggestion
        FMLCommonHandler.instance().bus().register(new HooksKey());
```